mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/StatefulContestScreenTest.kt

```diff
@@ -129,11 +114,9 @@ class StatefulContestScreenTest {
       authFacade.signInWithEmail("email@example.org", "password")
       val user = authFacade.currentUser.filterIsInstance<AuthenticatedUser>().first()
 
-      val strings =
-          rule.setContentWithLocalizedStrings {
-            ProvideFacades(authFacade, socialFacade, chessFacade, speechFacade, tournamentFacade) {
-              StatefulHome(user)
-            }
+      val (_, _, strings) =
+          rule.setContentWithTestEnvironment(store = store, auth = auth, timeProvider = time) {
```

Comment:
You should pass the `assets` here, and the `datastoreFactory`

Reply:
thx, the dataStore was not needed